spec/pageflow/file_types_spec.rb

```diff
@@ -51,9 +61,30 @@ def initialize(options)
       it 'raises exception if FileType is not found' do
         file_types = FileTypes.new([])
 
-        expect {
+        expect do
           file_types.find_by_collection_name!('image_files')
-        }.to raise_error(FileType::NotFoundError)
+        end.to raise_error(FileType::NotFoundError)
```

Comment:
Let's not keep following this style rule. I actually think `do ... end.to` reads a lot worse than with the multi line `{ }`. Rubocop has a setting for [`Style/BlockDelimiters` called `braces_for_chaining`](https://github.com/bbatsov/rubocop/blob/v0.39.0/config/default.yml#L205) which does exactly what I want, so I suggest we change the linter config file to use that. I know you already converted many instances to the `do ... end.to` style on the permissions branch. But even there both usages are still present (49 vs 51 occurrences), so I'd argue that eventually reverting the changes is just as much work as converting the remaining instances ;)